tests/modules/nf-core/bbmap/bbnorm/test.yml

```diff
@@ -0,0 +1,42 @@
+- name: bbmap bbnorm test_bbmap_bbnorm_se
+  command: nextflow run ./tests/modules/nf-core/bbmap/bbnorm -entry test_bbmap_bbnorm_se -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/bbmap/bbnorm/nextflow.config
+  tags:
+    - bbmap/bbnorm
+    - bbmap
+  files:
+    - path: output/bbmap/test.bbnorm.log
+      contains:
+        - f64935b85650bd1f3b2f8d8e77736cf4
+    - path: output/bbmap/test.fastq.gz
+      contains:
+        - 6b945a91d0e77c5ded35ff6fddf94b10
+    - path: output/bbmap/versions.yml
+        - e9ac278bca50f89b21b97e3b97415648
```

Comment:
I'm actually sure the values are md5sums, so you should have lines like:
```
    - path: ./output/gffread/genome.out.gtf
      md5sum: fcbf5744ed806e47768bc456fa043263
```
Use `contains` when the md5sum is not stable, i.e. fails in some tests. Common with gzipped files. An example:
```
contains: ["29829"]
```
The above will look for that string. Works also in gzipped files.
More examples below.